swift/test/Ice/adapterDeactivation/slice-plugin.json

```diff
@@ -1,3 +0,0 @@
-{
-  "sources": ["Test.ice"]
-}
```

Comment:
For this simple case no need to include a config anymore, we just build all `.ice` files in the target inputs.